Src/VsVimShared/Implementation/Misc/VsAdapter.cs

```diff
@@ -318,6 +318,36 @@ internal bool IsWatchWindowView(ITextView textView)
             return value.Equals(true) || value.Equals(1);
         }
 
+        internal bool IsTextEditorView(ITextView textView)
+        {
+            // Our fonts and colors won't work unless this view is in the "Text Editor"
+            // fonts and colors category.

```

Comment:
FWIW: Not having the fonts and colors work is less than ideal but it's not necessarily a deal breaker for me.